index.html

```diff
@@ -1,88 +1,114 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title>MARBLE</title>
-	<link href='https://fonts.googleapis.com/css?family=Roboto Condensed' rel='stylesheet'>
-	<link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
-	<link rel="stylesheet" type="text/css" href="styles/style.css">
-	
+    <head>
+        <title>Responsive</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href='https://fonts.googleapis.com/css?family=Roboto Condensed' rel='stylesheet'>
+        <link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <header>
+            <div class="container">
+                <div class="row">
+                    <div class="header-top">
+                        <h3 class="logo color-gray">marble</h3>
+                        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                            <i class="fa fa-bars fa-2x"></i>
+                        </a>
+                    </div>
+                    <nav class="bold nav-menu">
+                        <ul id="menu-responsive">
+                            <li><a href="#">home</a></li>
+                            <li><a href="#">about</a></li>
+                            <li><a href="#">jounal</a></li>
+                            <li><a href="#">service</a></li>
+                            <li><a href="#">feature</a></li>
+                            <li><a href="#">contact</a></li> 
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </header>
+        <div class="slideshow-container">
```

Comment:
Anh đang thấy slide show hiển thị khá to.
Để slideshow đẹp e nên chọn ảnh có kích thước đúng ý muốn nhé. Trong trường hợp em muốn dùng ảnh kích thước khác thì nên hiển thị bằng cách set back-ground image. Sau đó set kích thước cụ thể và dùng các thuộc tính như back-ground-size va back-ground-position để chỉnh sửa cho đẹp nhé
em tham khảo ở đây: https://codepen.io/tysonmatanich/pen/mewzxG